schemas/lrt/schema.json

```diff
@@ -0,0 +1,32 @@
+{
+    "schema": "Liquid Restaking Tokens (LRTs)",
+    "description": "Standard table definitions for LRTs (liquid restaking tokens)",
+    "version": "1.0.0-alpha",
+    "tables": [
+      {
+        "label": "EigenLayer Points Snapshot",
+        "tableName": "eigenlayer_points",
+        "aggregation": "daily",
+        "description": "User level snapshot of EigenLayer points earned by users in this protocol.",
+        "properties": {
+          "timestamp": {
+            "description": "The timestamp of the snapshot.",
+            "type": "number"
+          },
+          "protocol_name": {
+            "description": "The name of the LRT protocol.",
+            "type": "string"
+          },
+          "user_address": {
+            "description": "The address of the user earning points (lowercase only).",
+            "type": "string"
+          },
+          "eigenlayer_points": {
+            "description": "The amount of EigenLayer points earned by this user (decimal normalized).",
```

Comment:
this can just be "points", and then the description can be "The amount of points earned by the user, decimal normalized."